### PR TITLE
fix(runner): restore client verbosity

### PR DESCRIPTION
**Pull request by mhofman**
Also bump and unify task start timeouts

### CODE REVIEW

File: runner/lib/tasks/testnet.js [hunk 1]
```diff
@@ -338,7 +339,7 @@ ${chainName} chain does not yet know of address ${soloAddr}
 
     const chainEnv = Object.create(process.env);
     chainEnv.SLOGFILE = slogFifo.path;
-    // chainEnv.DEBUG = 'agoric';
```
Comment (michaelfig) [inline runner/lib/tasks/testnet.js:341]: Actually, this line should work, and give you more logs than VerboseDebugEnv.  The only reason you weren't seeing `swingset running` was that the timeout was too short.

File: runner/lib/tasks/testnet.js [hunk 2]
```diff
@@ -474,8 +475,9 @@ ${chainName} chain does not yet know of address ${soloAddr}
 
     const clientEnv = Object.create(process.env);
     clientEnv.SOLO_SLOGFILE = slogFifo.path;
+    clientEnv.DEBUG = VerboseDebugEnv;
```
Comment (michaelfig) [inline runner/lib/tasks/testnet.js:478]:
```suggestion
    clientEnv.DEBUG = 'agoric';
```

**Review by michaelfig — CHANGES_REQUESTED** [review]
Please use `DEBUG=agoric`, as that will give you maximum logs.

File: runner/lib/tasks/testnet.js [hunk 3]
```diff
@@ -35,6 +35,7 @@ const stateDir = '_agstate/agoric-servers';
 const profileName = 'testnet';
 const CLIENT_PORT = 8000;
 const clientStateDir = `${stateDir}/${profileName}-${CLIENT_PORT}`;
+const VerboseDebugEnv = 'SwingSet:vat,SwingSet:ls';
```
Comment (michaelfig) [inline runner/lib/tasks/testnet.js:38]:
```suggestion
const VerboseDebugEnv = 'agoric';
```

Comment (mhofman) [reply runner/lib/tasks/testnet.js:38]: I had copied the values from Agoric CLI, without knowing.

**Review by michaelfig — APPROVED** [review]
LGTM!

**Pull request merged**